core/src/main/resources/i18n/displayStrings_fr.properties

```diff
@@ -3477,7 +3477,7 @@ validation.sortCodeChars={0} doit être composer de {1} caractères.
 validation.bankIdNumber={0} doit être composer de {1} chiffres.
 validation.accountNr=Le numéro du compte doit comporter {0} chiffres.
 validation.accountNrChars=Le numéro du compte doit comporter {0} caractères.
-validation.btc.invalidAddress=L''adresse n''est pas correcte. Veuillez vérifier le format de l''adresse.
+validation.btc.invalidAddress=L'adresse n'est pas correcte. Veuillez vérifier le format de l'adresse.
```

Comment:
This is an example of the problems remaining in French and Italian, using apostrophe for accent is pervasive and doubling-up sometimes incorrect.

Reply:
I added a commit to fix a situation that I found whether substitution parameters are missing (or were removed at some time) but the code was still trying to to do the replacements. This certainly isn't the only way these have happened, however. That is another way these inconsistencies can happen, but really the correct fix in these cases is to modify the code to not pass the (unused) replacements. Really though, an even better solution for everyone (which should have been done at day-1) would be enhance Res.get() to perform the apostrophe double-up itself when replacement parameters are provided and make the language files consistent.